bin/disco_snapshot.py

```diff
@@ -60,6 +60,8 @@ def get_parser():
     parser_take_group.add_argument('--hostclass', dest='hostclasses', default=[], action='append', type=str)
     parser_take_group.add_argument('--ami', dest='amis', default=[], action='append', type=str)
     parser_take_group.add_argument('--volume-id', dest='volume_id', type=str)
+    parser_take_group.add_argument('--disk-usage', dest='disk_usage', required=False, type=str,
```

Comment:
The arg could be repeatable so we can pass in multiple `--tag` args to create multiple tags for the snapshot